package.json

```diff
@@ -54,6 +54,7 @@
     "babel": "6.5.2",
     "babel-plugin-add-module-exports": "0.1.2",
     "babel-plugin-syntax-decorators": "6.5.0",
+    "babel-plugin-transform-builtin-extend": "1.1.2",
```

Comment:
why is this change for along with the new entry on gobblefile.js?

Reply:
No, it is related to: `src/error.js`
here: `ReconnectFailedError extends LogentriesError`

Reply:
Is there no better way doing this? without introducing a new dependency?

Reply:
We have `babel-preset-es2015` which includes `babel-plugin-transform-es2015-classes`.

This last one supports partially and it recommends to use `babel-plugin-transform-builtin-extend` which also have limitations.

I am not sure.